core/trie/trie.go

```diff
@@ -6,23 +6,13 @@ import (
 	"fmt"
 	"math/big"
 	"strings"
+	"sync"
 
 	"github.com/NethermindEth/juno/core/crypto"
 	"github.com/NethermindEth/juno/core/felt"
 	"github.com/NethermindEth/juno/db"
 )
 
-// Storage is the Persistent storage for the [Trie]
-type Storage interface {
-	Put(key *Key, value *Node) error
-	Get(key *Key) (*Node, error)
-	Delete(key *Key) error
-
-	PutRootKey(newRootKey *Key) error
-	RootKey() (*Key, error)
-	DeleteRootKey() error
-}
-
```

Comment:
Can you please explain why this is being removed?

Reply:
From the commit message:

> It wasn't utilized anyways, no need to pay the virtualization cost